examples/training/classification/imagenet/basic_training.py

```diff
@@ -208,23 +283,22 @@ def augment(img, label):
 with strategy.scope():
     training_metrics = [metrics.CategoricalAccuracy()]
 
-
 """
 As a last piece of configuration, we configure callbacks for the method.
 We use EarlyStopping, BackupAndRestore, and a model checkpointing callback.
 """
 
 
 callbacks = [
-    callbacks.ReduceLROnPlateau(
-        monitor="val_loss", factor=0.1, patience=10, min_delta=0.001, min_lr=0.0001
-    ),
     callbacks.EarlyStopping(patience=20),
```

Comment:
These days I hardly see people use early-stop. Like, don't give up yet. 🤕

Reply:
> Yes - it doesn't restart. It goes globally from 0 to the initial rate, and then slowly lowers it from there (after an optional hold)

I meant if there was some specific improvement in your experiment against that.